veps/sig-network/passt-migration-proposal.md

```diff
@@ -0,0 +1,333 @@
+
+##### Failure Scenarios:
+
+passt-repair can fail or timeout if:
+- The Unix Domain Socket created by passt isn't found.
+- Communication with passt is delayed or unresponsive.
+
+Either scenario likely results in broken TCP connections for the guest VM, requiring reconnection.
+- virt-handler does not fail migration on passt-repair errors.
+- passt currently does not abort migration upon TCP_REPAIR failure.
+
+> [!NOTE]
+> Success of passt-repair doesn't guarantee seamless migration alone; sticky IPs from the network provider and CNI are essential.
+
+### passt network binding plugin 
+
+The binding of passt is implemented as a [network binding plugin](https://kubevirt.io/user-guide/network/network_binding_plugins/#network-binding-plugins).
+It comprises:
+- A [sidecar container](#Sidecar container) running in the virt-launcher pod. Its sole responsibility is to mutate the DomainXML.
+- A [CNI Plugin](#CNI plugin) that is invoked during pod setup/teardown. 
+- Configuration.
+Further details can be found in the [design document](https://github.com/kubevirt/community/blob/main/design-proposals/network-binding-plugin/network-binding-plugin.md#design-details).
+
+#### Sidecar container
+
+`vhost-user` enables high-performance communication between virtual machines and user-space processes, typically offloading VirtIO emulation to dedicated user-space processes (e.g., DPDK or passt). 
+It improves I/O performance through zero-copy mechanisms and reduced context-switching overhead. Additionally, it is required for seamless TCP migration.
+
+[vhost-user](#vhost-user) must be enabled in the DomainXML. Small code changes are required in passt sidecar [code](https://github.com/kubevirt/kubevirt/tree/release-1.5/cmd/sidecars/network-passt-binding). The format is described [here](https://libvirt.org/formatdomain.html#vhost-user-connection-with-passt-backend).
+
+#### CNI plugin 
+
+The [passt-cni](https://github.com/kubevirt/kubevirt/tree/release-1.5/cmd/cniplugins/passt-binding) process is called by `Multus` during primary network interface setup/removal. It runs two sysctl commands:
+1. Allow binding to all ports starting from 0.
+2. Set ping group range to virt-launcher's user ID.
+> [!NOTE] 
+> No changes are expected in the `passt` CNI code.
+
+### Build and Deployment
+
+#### Add passt-repair binary to virt-handler image
+
+The `passt-repair` binary will be added to the virt-handler container image installed from the `passt` RPM. 
+
+#### passt CNI plugin build and deployment manifests
+
+The [passt-binding directory](https://github.com/kubevirt/kubevirt/tree/release-1.5/cmd/cniplugins/passt-binding) currently includes a build system to create the CNI plugin binary; however, a container image build is missing.
+Therefore, a container image build will be added to CNAO.
```

Comment:
- Please add a link to CNAO.
- Please rephrase the "missing" container image. It is confusing for someone who reads it in the context of k/kubevirt. Nothing is missing, you are just asking to also cover the deployment step which requires to use a container image.
IMO the whole section should describe what is needed to deploy the needed, i.e. place the CNI binary and make the sidecar image available for consumption.
Then, you can add  that CNAO will automate these steps (add a link to it please).